### ISSUE TITLE
Changes to latexindent configuration requires to restart/reload VSCode

### ISSUE THREAD

**Issue by moi90** [James-Yu/LaTeX-Workshop#1857]
### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.

You visited the wiki?: Yes

## Describe the bug [Required]

The arguments set in `latex-workshop.latexindent.args` are not passed to `latexindent`.

### To Reproduce

In `.vscode/settings.json` configure:

```
{
    "latex-workshop.latexindent.args": [
        "-c",
        "%DIR%/",
        "%TMPFILE%",
        "-y=defaultIndent: '%INDENT%'",
        "-l"
    ]
}
```

In the `.tex` file, right-click, "Format Document".

### Expected behavior

latexindent is called with the configured parameters (most of all, -l in my case).

### LaTeX Workshop Output [Required]

Instead, the command remains the default:
```
[15:24:48] Formatting with arguments -c,/home/moi/Work/19-mc-paper/,/home/moi/Work/19-mc-paper/__latexindent_temp.tex,-y=defaultIndent: '    '
[15:24:48] Formatted /home/moi/Work/19-mc-paper/test.tex
```

## Desktop [Required]
 - OS: Ubuntu 18.04
 - VS Code version: 1.40.2
 - Extension version: 8.4.2

### Are you using LaTeX Workshop with VS Code Remote?

No

**Comment by trevorgunn** [comment]
I cannot reproduce this issue. Maybe this "In `.vscode/settings.yaml` configure:" is your problem. The settings should be in a `.json` file which is, by default `~/.config/Code/User/settings.json`. Check that when you open the settings editor in VSCode that the `latexworkshop.args` setting is what it should be.

**Comment by jlelong** [comment]
The settings definitely needs a `.json` extension. See [vscode's documentation](https://code.visualstudio.com/docs/getstarted/settings#_settings-file-locations) for the different places of configuration files.

**Comment by moi90** [comment]
Sorry, `.yaml` was a spelling error. It is actually `vscode/settings.json` and was generated by VS Code. Please re-open.

**Comment by jlelong** [comment]
OK, I get it. Currently, the configuration related to `latexindent` is read only once when the extension is loaded. Any further changes will only be taken into account after restarting/reloading VSCode. So the first `.tex` file you open (or more precisely its workspace) fixes the configuration once and for all.

I will change this.

**Comment by moi90** [comment]
Ah, I understand! I can confirm that the command line matches the configuration after restarting the IDE.

**Comment by jlelong** [comment]
It should be fixed by #1863. @moi90 If you know how to do it, can you check it works for you under Linux?

**Comment by moi90** [comment]
Sorry, I don't know how to install the development version of an extension.

**Comment by tecosaur** [comment]
@moi90 try this:

 1. Make sure you have `npm` and `git`
 2. Open a terminal and `cd` to some convenient location
 3. `https://github.com/James-Yu/LaTeX-Workshop.git`
 4. Open the created folder in vscode (`code ./LaTeX-Workshop`  might work)
 5. Press <kbd>F5</kbd>

This should open up the development version of the extension.

**Comment by jlelong** [comment]
Closed by #1863.
